machine/os/ubuntu.go

```diff
@@ -0,0 +1,37 @@
+package os
+
+import (
+	"strings"
+
+	"github.com/docker/machine/drivers"
+)
+
+func init() {
+	RegisterRuntime("ubuntu", &RegisteredRuntime{
+		Detect: UbuntuDetection,
+	})
+}
+
+func UbuntuDetection(d drivers.Driver) (*Runtime, error) {
+	out, err := sshCommand(d, "lsb_release -i | cut -f2 -d:")
```

Comment:
Oh cool.  @sthulb it looks like we want to match on `ID` field in `/etc/os-release`.  I tested for the file in Debian, boot2docker, and Fedora and it was present in all of them.  (I tested the Debian and Fedora ones in containers)